spec/models/offer_spec.rb

```diff
@@ -11,4 +11,13 @@
     offer = build(:offer, user: nil)
     expect(offer).to_not be_valid
   end
+
+  describe '#to_param' do
+  	let(:user) { build(:user) }
+  	let(:offer) { build(:offer, user: user) }
+
+		it 'should parameterize id' do
+			expect(offer.to_param).to eq("#{offer.id}-#{offer.name.parameterize}")
```

Comment:
This line sat in the controller spec, but I've moved it to the model spec as it tests a method on the `Offer` model